src/Microsoft.Diagnostics.ExtensionCommands/FindPointersInCommand.cs

```diff
@@ -0,0 +1,517 @@
+﻿using Microsoft.Diagnostics.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static Microsoft.Diagnostics.ExtensionCommands.MAddressCommand;
+using Microsoft.Diagnostics.DebugServices;
+using System.IO;
+using System.Diagnostics;
+
+namespace Microsoft.Diagnostics.ExtensionCommands
+{
+    [Command(Name = "findpointersin", Help="Finds pointers to the GC heap within the given memory regions.")]
+    public class FindPointersInCommand : ExtensionCommandBase
+    {
+        private const string VtableConst = "vtable for ";
+
+        public IModuleService ModuleService { get; set; }
```

Comment:
Now that my PR has been merged, all the services need the ServiceImport attribute on them.